editor/editor_inspector.cpp

```diff
@@ -4084,6 +4167,52 @@ void EditorInspector::_node_removed(Node *p_node) {
 	}
 }
 
+void EditorInspector::_configuration_info_changed(Object *p_object) {
+	if (object == p_object) {
+		// Only update the tree if the list of configuration info has changed.
+		if (_update_configuration_info()) {
+			update_tree_pending = true;
+		}
+	}
+
+	for (Variant child : find_children("", "EditorInspector", true, false)) {
+		Object::cast_to<EditorInspector>(child)->_configuration_info_changed(p_object);
+	}
```

Comment:
You could use `propagate_call()` instead.

Reply:
Fairly sure that only works for methods that are bound, since there's no way to pass it a callable. Can a method be bound without being visible to scripting, e.g. if underscore prefixed?

Reply:
Yes, you can prefix it with underscore.